Woof/Woof/View/ReusableViews/TextEditorWithPlaceholder.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+/// A view displaying TextEditor with a prompt text message for a user.
+struct TextEditorWithPlaceholder: View {
+    /// The text to be shown in the TextEditor.
```

Comment:
It doesn't clear what is TextEditor.
```suggestion
    /// The text to be shown in the text input field.
```

Please, review the rest of code.

Reply:
Thanks, fixed it.